sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/recognizePiiEntitiesForBatchInputForDomainFilter.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Are we missing a test case with showStats=true?

Reply:
We already have tests for showing the statistics and the statis is more for batch info but not a specific document result statistics. 
Those are the properties in the TextDocumentBatchStatistics
```
    private final int documentCount;
    private final int validDocumentCount;
    private final int invalidDocumentCount;
    private final long transactionCount;
```